mobile/cmp/header/AppMenuButton.ts

```diff
@@ -23,6 +23,9 @@ export interface AppMenuButtonProps extends MenuButtonProps {
     /** True to hide the Feedback item. */
     hideFeedbackItem?: boolean;
 
+    /** True to hide the Reload button. Defaulted to false. */
+    hideReloadItem: boolean;
```

Comment:
Tweaking now to make this optional - noticed that Toolbox threw a `tsc` warning once updated to the latest snap as it was not passing this prop (which I assume it fine not to pass)